roles/openshift_version/tasks/set_version_containerized.yml

```diff
@@ -2,6 +2,12 @@
 - set_fact:
     l_use_crio_only: "{{ openshift_use_crio_only | default(false) }}"
 
+- name: Validate openshift_release or openshift_image_tag is specified
+  fail:
+    msg: "openshift_release or openshift_image_tag must be specified for enterprise version of containerized installation."
+  when:
+  - (openshift_release is not defined and openshift_image_tag is not defined | bool) and deployment_type == 'openshift-enterprise'
```

Comment:
should this be done for any containerized installation?

Reply:
I considered it. But origin's image was not tagged as the version of `openshift_release`.

- enterprise (it has `v3.5`, `v3.6` and so on)
https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/openshift3/node

- origin (no `v1.5` or `v3.6` tag)
https://hub.docker.com/r/openshift/origin/tags/

So, if origin user specified the image with `openshift_release` like `v3.6`, the image will not be found. `openshift_version` or `openshift_image_tag` could be mandatory for them, but I wonder it is not necessary.